[2024-07-16T10:30:38.648+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_remote_jupyter_notebook.run_notebook manual__2024-07-16T10:30:36.400659+00:00 [queued]>
[2024-07-16T10:30:38.654+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_remote_jupyter_notebook.run_notebook manual__2024-07-16T10:30:36.400659+00:00 [queued]>
[2024-07-16T10:30:38.655+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-07-16T10:30:38.666+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): run_notebook> on 2024-07-16 10:30:36.400659+00:00
[2024-07-16T10:30:38.671+0000] {standard_task_runner.py:57} INFO - Started process 246 to run task
[2024-07-16T10:30:38.674+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'run_remote_jupyter_notebook', 'run_notebook', 'manual__2024-07-16T10:30:36.400659+00:00', '--job-id', '1534', '--raw', '--subdir', 'DAGS_FOLDER/run_notebook.py', '--cfg-path', '/tmp/tmpazxwayx8']
[2024-07-16T10:30:38.676+0000] {standard_task_runner.py:85} INFO - Job 1534: Subtask run_notebook
[2024-07-16T10:30:38.715+0000] {task_command.py:410} INFO - Running <TaskInstance: run_remote_jupyter_notebook.run_notebook manual__2024-07-16T10:30:36.400659+00:00 [running]> on host b473ec83033a
[2024-07-16T10:30:38.796+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_remote_jupyter_notebook' AIRFLOW_CTX_TASK_ID='run_notebook' AIRFLOW_CTX_EXECUTION_DATE='2024-07-16T10:30:36.400659+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-16T10:30:36.400659+00:00'
[2024-07-16T10:30:40.498+0000] {logging_mixin.py:149} INFO - Notebook executed and updated successfully on the server.
[2024-07-16T10:30:40.499+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-07-16T10:30:40.508+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=run_remote_jupyter_notebook, task_id=run_notebook, execution_date=20240716T103036, start_date=20240716T103038, end_date=20240716T103040
[2024-07-16T10:30:40.534+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-07-16T10:30:40.548+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
